### PR TITLE
Frontend V1.9.1 (Launch Sequence)

### PR DESCRIPTION
**Pull request by Jgerbrandt**
- Automatic Launch sequence
- Waits for telemetry from ignitors before opening MEV
- Ignitors disabled and only show state

### CODE REVIEW

File: RocketControlUnitGUI/src/routes/+page.svelte [hunk 1]
```diff
@@ -410,6 +410,43 @@
 		});
 	}
 
+	let wasLiveAtAnyPoint = false;
+
+	async function pollIgnitors() {
+		if (box1_display === 'LIVE' || box2_display === 'LIVE') {
+			return wasLiveAtAnyPoint = true;
```
Comment (cjchanx) [inline RocketControlUnitGUI/src/routes/+page.svelte:417]: Should do what we expect it to, but returning `=` is a little less clear than just assigning it

File: RocketControlUnitGUI/src/routes/+page.svelte [hunk 2]
```diff
@@ -410,6 +410,43 @@
...
+
+		await new Promise(resolve => setTimeout(resolve, 3500));
+
+		clearInterval(pollInterval);
+
+		if (wasLiveAtAnyPoint) {
+			await PB.collection('CommandMessage').create({
+				target: 'NODE_DMB',
+				command: 'RSC_IGNITION_TO_LAUNCH'
+			});
```
Comment (cjchanx) [inline RocketControlUnitGUI/src/routes/+page.svelte:444]: should we do the thing to send 3x with like 100ms delay?

**Review by cjchanx — APPROVED** [review]

File: RocketControlUnitGUI/src/routes/+page.svelte (hunk 1 above)
Comment (Jgerbrandt) [reply RocketControlUnitGUI/src/routes/+page.svelte]: done

File: RocketControlUnitGUI/src/routes/+page.svelte (hunk 2 above)
Comment (Jgerbrandt) [reply RocketControlUnitGUI/src/routes/+page.svelte]: done

**Review by cjchanx — APPROVED** [review]

**Pull request merged**